lib/data_uri/uri.rb

```diff
@@ -61,6 +61,6 @@ def self.build(arg)
     end
   end
 
-  @@schemes['DATA'] = Data
+  scheme_list['DATA'] = Data
```

Comment:
Perhaps this new improvement could also be used?
https://github.com/ruby/uri/commit/4346daac75b6b7772e67c81b1e0eab80c4f87034

Reply:
I made a Rails initializer with this content, which worked for me:

```ruby
require "data_uri"
require "data_uri/open_uri"

URI.register_scheme(:DATA, URI::Data)
```